Gopkg.toml

```diff
@@ -15,10 +15,8 @@ ignored = ["google.golang.org/appengine*"]
   name = "code.gitea.io/sdk"
 
 [[constraint]]
-#  branch = "master"
-  revision = "c74e08f039e56cef576e4336382b2a2d12d9e026"
+  branch = "master"
   name = "github.com/blevesearch/bleve"
-#Not targetting v0.7.0 since standard where use only just after this tag
```

Comment:
Near bottom of this file you can remove constraint on github.com/go-gitea/bolt"

Reply:
done.

Reply:
I think we should still fix revision to have predictable build

Reply:
@lafriks done.